Symfony/CS/Fixer/Contrib/NoEmptyCommentFixer.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class NoEmptyCommentFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens as $index => $token) {
+            if ($token->isGivenKind(T_COMMENT)) {
+                $this->fixComment($tokens, $index);
+            }
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'There should not be an empty comments.';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        // should be run after PhpdocToCommentFixer and before ExtraEmptyLinesFixer, TrailingSpacesFixer and WhitespacyLinesFixer.
+        return 2;
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index  T_COMMENT index
+     */
+    private function fixComment(Tokens $tokens, $index)
+    {
+        $content = $tokens[$index]->getContent();
+        if ('#' === $content[0]) {
+            if (preg_match('/^#\s*$/', $content)) {
```

Comment:
Please, use same delimiters in one class if possible (here, it is ;) ) - eg `{...}`